fpm/fpm.toml

```diff
@@ -14,12 +14,23 @@ tag = "v0.2"
 git = "https://github.com/urbanjost/M_CLI2.git"
 rev = "649075aceb97f997665a1a4656514fd2e9b4becc"
 
+[dependencies.fortran-intrinsic-manpages]
+git = "https://github.com/urbanjost/M_intrinsics.git"
+rev = "a758ebdd0487e8cab5b4894fbf375f87147c44b5"
```

Comment:
Why do we need to depend on this module? I see that it offers a helpful commandline interface to manual pages for Fortran, but how does this connect to fpm?

I would personally install the package with fpm and use it via the usual man interface than rather than have it as part of the package manager itself. Or am I missing something here?

Reply:
I believe the main advantage to integrating with fpm is that it is a cross-plaforn solution (_i.e._ it works on Windows). I think it's a neat feature. True the same functionality could be provided by a standalone executable package installable via `fpm`.

This comes down to what is in scope for `fpm`: is `fpm` just for fetching dependencies and building/testing/running or shall it be broader to include peripheral programmer tasks like documentation _etc._?

Perhaps as an immediate solution, and in order to move forward, we should prioritise the former and leave discussion of the latter for the long term; in which case we should remove this functionality from this PR for the time being. What are other people's thoughts?